### DISCUSSION TITLE
Using verovio to upgrade MEI versions

### DISCUSSION THREAD

**Discussion by craigsapp** [rism-digital/verovio#3403] [General]
In the Verovio README.me file, it states:

> Older versions of MEI are still supported by newer versions of Verovio. MEI files are internally upgraded when loaded into Verovio. This applies only to the features supported by Verovio.

does this allow updating (of features supported by Verovio), so that the command:

```bash
verovio -t mei file-old.mei -o file-new.mei
```

would have `file-new.mei` contain the updates to the most recent version of MEI?

**Comment by lpugin** [comment]
There is some infomation about it here https://book.verovio.org/toolkit-reference/input-formats.html#mei

Namely, Verovio will upgrade the features previously supported and that have change in MEI. It will not do anything for features not supported or not supported in a previous version.

This will be updated once MEI 5.0 is out.
